app/models/prison.rb

```diff
@@ -49,11 +49,14 @@ def confirm_by(today = Time.zone.today)
   end
 
   def validate_visitor_ages_on(target, field, ages)
+    return if ages.blank?
     adults, _children = ages.partition { |a| adult?(a) }.map(&:length)
-    if adults > MAX_ADULTS
+    # For simplicity’s sake, this operates on the possibly unsafe assumption
+    # that the first visitor is always going to be the requesting visitor.
+    if ages.first < REQUESTING_VISITOR_MIN_AGE
```

Comment:
Sorting by age would defeat the purpose.  If the *requesting* (first) visitor is 12, but there is some over 18 in the group, then the request would be allowed.  This is to ensure that the person making the request is over 18, not that the group has an adult in it. I'm guessing there is a legal requirement regarding the minimum age of the requestor?  @kyliemulholland?